newdle/client/src/UserMenu.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {Dropdown, Icon} from 'semantic-ui-react';
+import Gravatar from 'react-gravatar';
+
+import './UserMenu.scss';
+
+const defaultUIAvatarsArgs = [
+  40, // size
+  'f0e9e9', // background
+  '8b5d5d', // color
+  2, // length
+  0.5, // font size
+  true, // rounded
+];
+
+export default function UserMenu({user}) {
+  const {email, fullName} = user;
+  const gravatarFallback = fullName
+    ? encodeURI(`//ui-avatars.com/api/${fullName}/${defaultUIAvatarsArgs.join('/')}`)
```

Comment:
I think it's enough to pass the first character of the first/last name here; No need to expose more data than needed to an external service.

Reply:
Yep, definitely ;)